cmd/tableprinters/audit.go

```diff
@@ -23,18 +23,23 @@ func (t *TablePrinter) AuditTable(data []*models.V1AuditResponse, wide bool) ([]
 			statusCode = fmt.Sprintf("%d", trace.StatusCode)
 		}
 		row := []string{
-			time.Time(trace.Timestamp).Format(time.StampMilli),
+			time.Time(trace.Timestamp).Format(time.DateTime),
 			trace.Rqid,
+			trace.Component,
 			trace.Detail,
 			trace.Path,
 			statusCode,
-			trace.Tenant,
 			trace.User,
-			trace.Phase,
 		}
 
 		if wide {
-			row = append(row, trace.Body)
+			row = append(row, trace.Tenant)
+
+			body := trace.Body
+			if len(body) > 40 {
+				body = body[:37] + "..."
+			}
```

Comment:
We have this functionality in the metal-lib here: https://github.com/metal-stack/metal-lib/blob/master/pkg/genericcli/truncate.go

Reply:
Thanks! Opened PR: https://github.com/metal-stack/metalctl/pull/198